lib/uuid.js

```diff
@@ -0,0 +1,13 @@
+import seededRandom from 'seed-random'
```

Comment:
This shouldn't be needed anymore, right, since you're using the higher-order-components package instead. Same for `lib/uniqueComponentName`.